com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandard.shader

```diff
@@ -184,11 +184,57 @@ Shader "Graphics Tools/Standard"
 
             #pragma shader_feature_local_fragment _CLIPPING_BORDER
 
+            // From SimpleLit SubShader "ForwardLit"...
+            // ...
+            // -------------------------------------
+            // Universal Pipeline keywords
+            #pragma multi_compile _ _MAIN_LIGHT_SHADOWS
+            #pragma multi_compile _ _MAIN_LIGHT_SHADOWS_CASCADE
+            #pragma multi_compile _ _ADDITIONAL_LIGHTS_VERTEX _ADDITIONAL_LIGHTS
+            #pragma multi_compile_fragment _ _ADDITIONAL_LIGHT_SHADOWS
+            #pragma multi_compile_fragment _ _SHADOWS_SOFT
+            #pragma multi_compile _ LIGHTMAP_SHADOW_MIXING
+            #pragma multi_compile _ SHADOWS_SHADOWMASK
+            #pragma multi_compile_fragment _ _SCREEN_SPACE_OCCLUSION
+
             #include_with_pragmas "GraphicsToolsStandardProgram.hlsl"
 
             ENDHLSL
         }
 
+        // From com.unity.render-pipelines.universal@10.9.0\Shaders\SimpleLit.shaderSimpleLit.shader
+        Pass
+        {
+            Name "ShadowCaster"
+            Tags{"LightMode" = "ShadowCaster"}
+
+            ZWrite On
+            ZTest LEqual
+            ColorMask 0
+            Cull[_Cull]
+        
+            HLSLPROGRAM
+            #pragma exclude_renderers gles gles3 glcore
+            #pragma target 4.5
+
+            // -------------------------------------
+            // Material Keywords
+            #pragma shader_feature_local_fragment _ALPHATEST_ON
+            #pragma shader_feature_local_fragment _GLOSSINESS_FROM_BASE_ALPHA
+
+            //--------------------------------------
+            // GPU Instancing
+            #pragma multi_compile_instancing
+            #pragma multi_compile _ DOTS_INSTANCING_ON
+
+            #pragma vertex ShadowPassVertex
+            #pragma fragment ShadowPassFragment
+
+            #include "Packages/com.unity.render-pipelines.universal/Shaders/SimpleLitInput.hlsl"
+            #include "Packages/com.unity.render-pipelines.universal/Shaders/ShadowCasterPass.hlsl"
```

Comment:
You might need to fork this and adjust the input and fragment shader since we don't have properites like  `_BaseMap` or `_BaseColor`. And we do have properties like `_Fade.`

Reply:
So this is pretty strange. The SimpleLit/Lit URP shaders don't appear to support alpha/transparent shadow casting (2021.3). Which is why they need access to _BaseMap, theoretically. 

In effect - it "just works" without having to fork a bunch of files (those includes point to includes...)

FWIW I'm able to common out _ALPHATEST_ON and DOTS_INSTANCING_ON, sliming this down slightly.

![image](https://user-images.githubusercontent.com/538256/213013006-69bc5f81-7825-4b1b-8633-4181b3e1ffe2.png)

Reply:
Looks like transparent shadow casting [isn't supported](https://forum.unity.com/threads/urp-lit-shader-will-not-receive-shadows-while-transparency-is-enabled.790193/) (not too surprising) but you do need the base map for alpha tested shadows: 
![image](https://user-images.githubusercontent.com/13305729/213016953-91492294-1873-47c6-8edc-8f9b99de22c9.png)

Reply:
Working now. Still have to get round corners working.